repository/repository/3.root.json

```diff
@@ -133,4 +133,4 @@
 		"spec_version": "1.0",
 		"version": 3
 	}
-}
+}
```

Comment:
Removing the newline from the end of the file changes the digest of the metadata file. I wonder what's causing this 🤔

Reply:
A go-tuf update we did 16 days ago? Not sure.

Reply:
Is this safe to merge?

Reply:
seems like all previous update snapshot & timestamps triggered that. Can't understand what changed.